notify/src/subscription/single.rs

```diff
@@ -308,11 +556,27 @@ impl Subscription for UtxosChangedSubscription {
     }
 
     fn active(&self) -> bool {
-        self.active
+        self.state().active()
+    }
+
+    fn scope(&self, context: &SubscriptionContext) -> Scope {
+        // TODO: consider using a provided prefix
+        UtxosChangedScope::new(self.data().to_addresses(Prefix::Mainnet, context)).into()
```

Comment:
`Prefix::Mainnet`: the prefix should be dynamic and based on actual network. Remove TODO after fixing